draft-packages/menu/KaizenDraft/Menu/components/MenuContent.module.scss

```diff
@@ -42,6 +42,10 @@ $side-padding: 1/2 * $ca-grid;
   align-items: center;
   flex: 0 0 100%;
   box-sizing: border-box;
+  background: none;
+  border: none;
+  width: 100%;
+  text-align: left;
```

Comment:
The `text-align` was previously being inherited from `.menuContent`, which handles RTL. If we're adding it here could we do the same?

See https://github.com/cultureamp/kaizen-design-system/pull/785/files#diff-270ef25388a53fab2ab6511fe5ea3f26R21